fabtests/common/shared.c

```diff
@@ -1103,6 +1117,10 @@ int ft_retrieve_conn_req(struct fid_eq *eq, struct fi_info **fi)
 		return ret;
 	}
 
+	if (ft_verify_info(fi_pep, entry.info)) {
+		return -FI_EOTHER;
```

Comment:
nit: Since ft_verify_info returns an error code, it would be better to capture that return code and pass it through, rather than converting it to FI_EOTHER.

Reply:
Would this be better:

if (entry.info != NULL){
        ret = ft_verify_info(fi_pep, entry.info);
        return ret;
}

Reply:
entry.info needs to be non-null here.  It's better to set ret = ft_verify_info(...) and return ret.